lib/elixir/test/elixir/code_test.exs

```diff
@@ -126,40 +153,48 @@ defmodule CodeTest do
+    assert catch_error(
+             Code.string_to_quoted!(":there_is_no_such_atom", existing_atoms_only: true)
+           ) == :badarg
   end
 
   test "string_to_quoted/2 with :formatter_metadata wraps literals in blocks" do
     string_to_quoted = &Code.string_to_quoted!(&1, formatter_metadata: true)
 
-    assert string_to_quoted.(~s("one")) ==
-           {:__block__, [format: :string, line: 1], ["one"]}
-    assert string_to_quoted.("'one'") ==
-           {:__block__, [format: :charlist, line: 1], ['one']}
-    assert string_to_quoted.("?é") ==
-           {:__block__, [original: '?é', line: 1], [233]}
-    assert string_to_quoted.("0b10") ==
-           {:__block__, [original: '0b10', line: 1], [2]}
-    assert string_to_quoted.("12") ==
-           {:__block__, [original: '12', line: 1], [12]}
-    assert string_to_quoted.("0o123") ==
-           {:__block__, [original: '0o123', line: 1], [83]}
-    assert string_to_quoted.("0xEF") ==
-           {:__block__, [original: '0xEF', line: 1], [239]}
-    assert string_to_quoted.("12.3") ==
-           {:__block__, [original: '12.3', line: 1], [12.3]}
-    assert string_to_quoted.("nil") ==
-           {:__block__, [line: 1], [nil]}
-    assert string_to_quoted.(":one") ==
-           {:__block__, [line: 1], [:one]}
-    assert string_to_quoted.("[1]") ==
-           {:__block__, [end_line: 1, line: 1], [[{:__block__, [original: '1', line: 1], [1]}]]}
-    assert string_to_quoted.("{:ok, :test}") ==
-           {:__block__, [end_line: 1, line: 1], [{{:__block__, [line: 1], [:ok]}, {:__block__, [line: 1], [:test]}}]}
-    assert string_to_quoted.(~s("""\nhello\n""")) ==
-           {:__block__, [format: :bin_heredoc, line: 1], ["hello\n"]}
-    assert string_to_quoted.("'''\nhello\n'''") ==
-           {:__block__, [format: :list_heredoc, line: 1], ['hello\n']}
+    assert string_to_quoted.(~s("one")) == {:__block__, [format: :string, line: 1], ["one"]}
+    assert string_to_quoted.("'one'") == {:__block__, [format: :charlist, line: 1], ['one']}
+    assert string_to_quoted.("?é") == {:__block__, [original: '?é', line: 1], [233]}
+    assert string_to_quoted.("0b10") == {:__block__, [original: '0b10', line: 1], [2]}
+    assert string_to_quoted.("12") == {:__block__, [original: '12', line: 1], [12]}
+    assert string_to_quoted.("0o123") == {:__block__, [original: '0o123', line: 1], [83]}
+    assert string_to_quoted.("0xEF") == {:__block__, [original: '0xEF', line: 1], [239]}
+    assert string_to_quoted.("12.3") == {:__block__, [original: '12.3', line: 1], [12.3]}
+    assert string_to_quoted.("nil") == {:__block__, [line: 1], [nil]}
+    assert string_to_quoted.(":one") == {:__block__, [line: 1], [:one]}
+
+    assert string_to_quoted.("[1]") == {
+             :__block__,
+             [end_line: 1, line: 1],
+             [[{:__block__, [original: '1', line: 1], [1]}]]
+           }
```

Comment:
This one would need to break into the previous line. something like:

```elixir
args = [[{:__block__, [original: '1', line: 1], [1]}]]
assert string_to_quoted.("[1]") ==
       {:__block__, [end_line: 1, line: 1], block}
```

Do the same below and please format the file again.

Reply:
Fixed.